.drone.yml

```diff
@@ -4,17 +4,80 @@ workspace:
 
 pipeline:
   test:
-    image: golang:1.8
+    image: golang:1.9
     commands:
       - echo starting
       - cd drone
       - go test
 
-  build:
-    image: golang:1.8
-    commands: sh .drone.sh
+  build_linux_amd64:
+    image: golang:1.9
+    pull: true
+    group: build
+    environment:
+      - GOOS=linux
+      - GOARCH=amd64
+      - CGO_ENABLED=0
+    commands:
+      - go build -v -ldflags "-X main.version=${DRONE_TAG##v}" -a -o release/linux/amd64/drone github.com/drone/drone-cli/drone
+      - tar -cvzf release/drone_linux_amd64.tar.gz   -C release/linux/amd64   drone
+
+  build_linux_arm64:
+    image: golang:1.9
+    pull: true
+    group: build
+    environment:
+      - GOOS=linux
+      - GOARCH=arm64
+      - CGO_ENABLED=0
+    commands:
+      - go build -v -ldflags "-X main.version=${DRONE_TAG##v}" -a -o release/linux/arm64/drone github.com/drone/drone-cli/drone
+      - tar -cvzf release/drone_linux_arm64.tar.gz   -C release/linux/arm64   drone
+
+  build_linux_arm:
+    image: golang:1.9
+    pull: true
+    group: build
+    environment:
+      - GOOS=linux
+      - GOARCH=arm
+      - CGO_ENABLED=0
+      - GOARM=7
+    commands:
+      - go build -v -ldflags "-X main.version=${DRONE_TAG##v}" -a -o release/linux/arm/drone github.com/drone/drone-cli/drone
+      - tar -cvzf release/drone_linux_arm.tar.gz     -C release/linux/arm     drone
+
+  build_windows:
```

Comment:
And if we are really picky this should also get an _amd64 :P

Reply:
lol yeah probably right